monix-catnap/shared/src/main/scala/monix/execution/package.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2014-2019 by The Monix Project Developers.
+ * See the project homepage at: https://monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix
+
+import cats.Contravariant
+
+import scala.concurrent.ExecutionContext
+
+package object execution {
```

Comment:
This is the hacky bit. It might seem innocent, but note that it's actually in the catnap subproject.

Reply:
This is an awesome hack if it allows us to keep the instances comfortable while removing the Cats-Effect dependency from `monix-execution`.